runtime/ts/slot-dom-consumer.ts

```diff
@@ -262,12 +260,6 @@ export class SlotDomConsumer extends SlotConsumer {
     return node[name] || node.getAttribute(name);
   }
 
-  _validateSubId(providedSlotSpec, subId) {
-    assert(!this.subId || !subId || this.subId === subId, `Unexpected sub-id ${subId}, expecting ${this.subId}`);
```

Comment:
`this.subId` is indeed not set, but verifying that `subId` matches the `providedSlotSpec.isSet` is kinda nice, no? doesn't need its own helper method of course, maybe just an assert?

Reply:
Yes, it's good to keep it. Re-added.